File: src/main/java/com/github/tonybaines/java/Loops.java
package com.github.tonybaines.java;

import java.util.ArrayList;
import java.util.List;

public class Loops {
    static List<String> rainbow = new ArrayList<>() {{
        add("red");
        add("orange");
        add("yellow");
        add("green");
        add("blue");
        add("indigo");
        add("violet");
    }};


    public static void main(String [] args) {
        for (String colour : rainbow) {
            System.out.println(colour);
        }
    }
}

File: src/main/kotlin/com/github/tonybaines/kotlin/basics/Loops.kt
package com.github.tonybaines.kotlin.basics

object Loops {
    val rainbow = listOf("red", "orange", "yellow", "green", "blue", "indigo", "violet")

    @JvmStatic
    fun main(args: Array<String>) {
        for (colour in rainbow) {
            println(colour)
        }
    }
}